cinema/serializers.py

```diff
@@ -1 +1,105 @@
-# write serializers here
+from rest_framework import serializers
+from rest_framework.serializers import ModelSerializer
+
+from cinema.models import Genre, Actor, Movie, MovieSession, CinemaHall
+
+
+class GenreSerializer(ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class ActorSerializer(ModelSerializer):
+
+    class Meta:
+        model = Actor
+        fields = (
+            "id",
+            "first_name",
+            "last_name",
+            "full_name"
+        )
+
+
+class CinemaHallSerializer(ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = (
+            "id",
+            "name",
+            "rows",
+            "seats_in_row",
+            "capacity"
+        )
+
+
+class MovieSerializer(ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = (
+            "id",
+            "title",
+            "description",
+            "duration",
+            "genres",
+            "actors",
+        )
+
+
+class MovieListSerializer(MovieSerializer):
+    genres = serializers.SlugRelatedField(
+        slug_field="name",
+        many=True,
+        read_only=True
+    )
+    actors = serializers.SlugRelatedField(
+        slug_field="full_name",
+        many=True,
+        read_only=True
+    )
+
+
+class MovieDetailSerializer(MovieSerializer):
+    genres = GenreSerializer(many=True, read_only=True)
+    actors = ActorSerializer(many=True, read_only=True)
+
+
+class MovieSessionSerializer(ModelSerializer):
+
+    class Meta:
+        model = MovieSession
+        fields = (
+            "id",
+            "show_time",
+            "movie",
+            "cinema_hall",
+        )
+
+
+class MovieSessionListSerializer(MovieSessionSerializer):
+    movie_title = serializers.CharField(source="movie.title",
```

Comment:
just an advice, use the same style as below

Reply:
corrected